composer.json

```diff
@@ -28,29 +28,23 @@
     },
     "sort-packages": true
   },
-  "extra": {
-    "downloads": {
-      "phpstan": {
-        "url": "https://github.com/phpstan/phpstan/releases/latest/download/phpstan.phar",
-        "path": "vendor/bin/phpstan",
-        "type": "phar"
-      }
-    }
-  },
   "autoload": {
     "psr-4": {
       "Google\\Web_Stories\\": "includes"
     }
   },
   "autoload-dev": {
     "psr-4": {
-      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes",
-      "PHPStan\\WordPress\\": "tests/phpstan/src"
+      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes"
     }
   },
   "prefer-stable": true,
   "scripts": {
     "phpcs": "phpcs",
-    "phpstan": "phpstan analyse --memory-limit=512M"
+    "phpstan": [
+      "mkdir php7 || echo",
```

Comment:
`php7.1` would be more correct I assume?

Still a bit generic though. Perhaps it could be prefixed with `composer-` or something.

Reply:
Actually there directory's name is `tools` (from PHIVE)

Reply:
Don't think I follow... `mkdir php7 || echo` to me says the directory's name is `php7`

Reply:
Yes, it is written in `composer.json` but the one can call a directory full of tools `tools` :)